### DISCUSSION TITLE
[Feature Request] Order for new entry when using Simple Task Editor

### DISCUSSION THREAD

**Discussion by sensen** [standardnotes/forum#2537] [Suggestions]
## Current implementation and background

Using Simple Task Editor, when adding a new entry, it is placed at the top of `Open Tasks` list. When adding tasks to be done in chronological order, it gets really troublesome having to rearrange everything.

## Expectation

An option, to put newly added task at the bottom of `Open Tasks` list.

**Comment by atmoio** [comment]
Definitely subjective. And no options menu currently to support preferences. But something to keep in mind for sure.

**Comment by sensen** [comment]
Is there any improvement to Simple Task Editor in the roadmap? @mobitar

**Comment by atmoio** [comment]
Nothing set in stone, but I was thinking it would be nice to have groups/categories within tasks. I use this editor extensively but create a new note for every context. Would be nice to have one large mega todo list with groups. I'm playing around with this on the `groups` branch, but still work to be done. https://github.com/sn-extensions/simple-task-editor/tree/groups (probably not going to actively resume work on this for a few weeks/months).

**Comment by EweSparky** [comment]
> Would be nice to have one large mega todo list with groups

I'd love to see this. Workaround for now is either multiple lists or prefixing todos with `Sub-group Name:`. Multiple lists don't always seem like the best option when a project has many sub-groups of only a couple of todo items each.

**Comment by JaspalSuri** [comment]
Noted your +1! Thanks for that tip as well!
